Makefile

```diff
@@ -3,7 +3,9 @@
 # Licenses: GPL2
 
 CC ?= cc
-WARNING = -Wall
+WARNING = -Wall -Wextra -D_FORTIFY_SOURCE=2 -fstack-protector-strong -Wformat -Wformat-security -Wdate-time -Wno-unused-parameter
```

Comment:
```
warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
  382 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
```

`_FORTIFY_SOURCE` warnings can be resolved by adding one _optimization level_ ; such as below where `OPTIM_LVL=` is setting one level for the whole project build:
```
make WARNING='-Wall -Wextra -D_FORTIFY_SOURCE=2 -fstack-protector-strong -Wformat -Wformat-security -Wdate-time -Wno-unused-parameter -Wunreachable-code -Wstrict-prototypes -Wformat-truncation' OPTIM_LVL=1 clean all
```
But this implies to set a default optimization level in the `Makefile`; things that CoreFreq let the user choose.